src/utils/bootstrap.js

```diff
@@ -63,13 +58,16 @@ export function bootstrapStore(client: any, { services, toolboxActions }: any) {
 }
 
 export function bootstrapWorkers() {
+  const workerPath = isDevelopment()
```

Comment:
nit: since isDevelopment() = !isFirefoxPanel(), it could be clearer to have only one test.